### ISSUE TITLE
[Bug] API is sending token as object

### ISSUE THREAD

**Issue by gustavorino** [mondaycom/monday-sdk-js#33]
Npm package monday-sdk-js@0.1.0

I'm trying the following code

```
monday.setToken("token");

monday
  .api("query { me { id, name } }", {
    token,
  })
  .then((res) => console.log(res));
```

And when I open the network tab I see this

![image](https://user-images.githubusercontent.com/4224712/89232350-a612b300-d5bd-11ea-9d66-ccf0c8798830.png)

Checking the code on client.js, I found this line : 

```
 return mondayApiClient.execute(params, {token} );
```

However, the execute function receives the token as String, not Object

```
async function execute(data, token, options = {}) {
... 
let response = await apiRequest(fullUrl, data, token, options);
```

**Comment by joeshonm** [comment]
I am also getting the same result. I'm guessing that's why the authentication is failing.

**Comment by vladmonday** [comment]
Hey @gustavorino, @joeshonm 
Thanks for reporting it, the issue was solved in the latest release 0.1.1, just update your dependency to the new version and the issue should be fixed

Thanks!
